packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -273,7 +279,7 @@ export class WalletService implements IWalletService {
   }
 
   static handleIncomingNotifications(notification, cb) {
-    cb = cb || function() {};
+    cb = cb || function () { };
```

Comment:
A space before the () has not been our lint convention (except before an `async` keyword)